.semaphoreci/test_10_ceph.sh

```diff
@@ -12,7 +12,7 @@ cd "$GIT_ROOT/integration_tests"
 # http://docs.ceph.com/docs/master/releases/schedule/
 
 ./docker_test_run.py \
-    --docker-image "ceph/daemon:latest-luminous" \
+    --docker-image "ceph/daemon:latest-mimic" \
```

Comment:
Phoey, this is actually the same SHA as `ceph/daemon`.  So it's testing the same thing twice.